subsys/bluetooth/host/Kconfig

```diff
@@ -335,6 +335,17 @@ config BT_SCAN_WITH_IDENTITY
 	  disclosing local identity information. However, if the use case
 	  requires disclosing it then enable this option.
 
+config BT_DEVICE_NAME_STORAGE
+	int "Size in bytes for device name"
+	default BT_RX_BUF_LEN
+	default BT_L2CAP_RX_MTU if BT_HCI_ACL_FLOW_CONTROL
```

Comment:
These defaults seem a bit odd to me. I'd make the default something that will always fit in full in the scan response data, meaning 28 (if I calculated correctly), or then what will always fit within the default ATT MTU.

Reply:
Right, I was thinking only in terms of ATT MTU but since we may also advertise that should be the default. We may also need to truncate and use the short time in the advertisement if the name is set to be bigger than that.